install_files/securedrop-app-code/DEBIAN/control

```diff
@@ -6,5 +6,5 @@ Homepage: https://securedrop.org
 Package: securedrop-app-code
 Version: 0.7.0~rc1
 Architecture: amd64
-Depends: python-pip,apparmor-utils,gnupg2,haveged,python,python-pip,secure-delete,sqlite,apache2-mpm-worker,libapache2-mod-wsgi,libapache2-mod-xsendfile,redis-server,supervisor,securedrop-keyring,securedrop-config,libjpeg-dev
+Depends: python-pip,apparmor-utils,gnupg2,haveged,python,python-pip,secure-delete,sqlite3,apache2-mpm-worker,libapache2-mod-wsgi,libapache2-mod-xsendfile,redis-server,supervisor,securedrop-keyring,securedrop-config,libjpeg-dev
```

Comment:
This change will be a problem for existing installs due to #3376 - is it strictly necessary?

Reply:
Sort of. The docs say that using the `sqlite3 $path_to_db .database` command is the most future-proof way to create a database file. That said, right now using `touch` does seem to work, but that goes against what the developers say we should do.

Reply:
Aslo use of `sqlite` to do this creates a database file that is incompatible with version 3 and causes errors about being unable to open the file.

Reply:
Recapping gitter comments/discussions for posterity:

Directly using `sqlite` to create a database will create a version 2.x database. This version is *totally incompatible* with version 3 (see the sqlite project's history of database file format changes [here](https://www.sqlite.org/formatchng.html)).

Since at least SecureDrop 0.3, we have used `db.create_all()` to create databases in production, and this method does create a version 3 database. Thus no 2->3 migration issues should be encountered. This logic is tested via the 0.7->0.8 upgrade path testing.